lms/resources/_js_config/__init__.py

```diff
@@ -418,6 +419,35 @@ def _config(self):
 
         return config
 
+    def _get_product_info(self):
+        product = self._request.product
+
+        product_info = {
+            "family": product.family,
+            "features": {
+                "groups": self._request.product.settings.groups_enabled,
+            },
+            # List of API endpoints we proxy for this product
+            "api": {},
+        }
+
+        # This should be abstracted as "lms_course_id" or similar in either
+        # LTIParams or as part of product.
+        api_course_id = self._request.lti_params.get(
+            "custom_canvas_course_id"
+        ) or self._request.lti_params.get("context_id")
```

Comment:
^ This section should be nested inside the if below. We don't use the value outside the if